onnxruntime/featurizers_ops/cpu/normalize_transformer.cc

```diff
@@ -58,6 +58,7 @@ struct NormalizeTransformerImpl {
       std::copy(result.cbegin(), result.cend(), output_data);
       output_data += row_size;
     }
+    transformer.flush(callback);
```

Comment:
>transformer.flush(callback); [](start = 4, length = 28)

Here, if the callback is invoked and result contains some data, it would be discarded?